From f602ef9a7de97e4d9fd016b2aaa08ec5b4f86710 Mon Sep 17 00:00:00 2001
From: Olliver Schinagl <oliver@schinagl.nl>
Date: Mon, 15 Apr 2019 09:47:34 +0200
Subject: [PATCH 1/3] unittests/test_lib: Include proper header for _IOC_SIZE

The macro _IOC_SIZE is not part of sys/ioctl.h but lives in asm/ioctl.h
so we should include the proper header. If we do not, some systems
complain during linking that they cannot find the symbol _IOC_SIZE()
which was not expanded by the pre-compiler.

Signed-off-by: Olliver Schinagl <oliver@schinagl.nl>
---
 tests/unittests/test_lib.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/unittests/test_lib.h b/tests/unittests/test_lib.h
index 7a6a003..cd94d9f 100644
--- a/tests/unittests/test_lib.h
+++ b/tests/unittests/test_lib.h
@@ -1,3 +1,4 @@
+#include <asm/ioctl.h>
 #include <stdarg.h>
 #include <setjmp.h>
 #include <stddef.h>
-- 
2.20.1

